grub-efi/Dockerfile.aarch64

```diff
@@ -1,7 +1,7 @@
-FROM debian:10
+FROM debian:11
```

Comment:
Generally I would agree that having a commit for updating the base image and another one to update the GRUB version is better, but I gave it a quick spin locally and I couldn't build the previous GRUB version with `debian:11` base image so the first commit is in itself broken.

I suggest we just merge your two commits, mentioning the image update in the commit message, and don't investigate it further.